src/commands/context/list.ts

```diff
@@ -0,0 +1,47 @@
+/*********************************************************************
+ * Copyright (c) 2020 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import { Command } from '@oclif/command'
+import { cli } from 'cli-ux'
+
+import { CheServerLoginManager } from '../../api/che-login-manager'
+
+export default class List extends Command {
+  static description = 'shows all existing login sessions'
```

Comment:
`Describe one or many contexts`?

Reply:
To me your variant is not clear. I'd like to stick to my version, unless you insist on the correction.